### PR TITLE
Ask user before deleting mongo database

### PR DESCRIPTION
**Pull request by firyx**
# PULL REQUEST

## Overview

Playbook `portal-setup-following` deletes and resets MongoDB database when it doesn't run correctly (e.g. the playbook can't connect to Mongo shell using defined admin credentials).

This PR asks user if it is ok to delete, reset MongoDB database or if he wants to fix it manually e.g. in case there are production data in the database.

## Checklist
Review and complete the checklist to ensure that the PR is complete before assigned to an approver.
 - [x] All new methods or updated methods have clear docstrings
 - [x] Testing added or updated for new methods
 - [x] Verify if any changes impact the WebPortal Health Checks
 - [x] Approriate documentation updated
 - [x] Changelog file created

### CODE REVIEW

**Comment by ro-tex** [comment]
Can we be even more defensive and always zip the existing `db` directory and keep the timestamped zip under `docker/data/mongo/restore_points` or something like that? If the DB is empty those zips will take almost no space at all and it's possible that they will save an operator who was in a hurry and just said "yes" to all questions without really reading them.

**Review by ro-tex — APPROVED** [review]

File: playbooks/tasks/portal-role-task-mongo-setup-docker-service.yml [hunk 1]
```diff
@@ -38,6 +38,28 @@
 
 # Reset mongo service if it is not running correctly
 - block:
+    - name: Ask user if it is ok to delete, reset MongoDB database
```
Comment (kwypchlo) [inline playbooks/tasks/portal-role-task-mongo-setup-docker-service.yml:41]:
```suggestion
    - name: Ask user if it is ok to delete and reset MongoDB database
```

File: playbooks/tasks/portal-role-task-mongo-setup-docker-service.yml [hunk 2]
```diff
@@ -38,6 +38,28 @@
...
+          it is safe to continue (MongoDB database will be deleted and reset).
+          
+          If you have production data in your database
+          you should fix this issue manually and rerun this playbook.
+          
+          Do you want to reset MongoDB database on {{ inventory_hostname }} (y/n)?
+      register: reset_db_result
+      delegate_to: localhost
+
+    - name: Stop the playbook if the user doesn't want to delete, reset MongoDB database
```
Comment (kwypchlo) [inline playbooks/tasks/portal-role-task-mongo-setup-docker-service.yml:56]:
```suggestion
    - name: Stop the playbook if the user doesn't want to delete and reset MongoDB database
```

**Review by kwypchlo — APPROVED** [review]
looks good, minor improvement in copy

**Comment by firyx** [comment]
> Can we be even more defensive and always zip the existing `db` directory and keep the timestamped zip under `docker/data/mongo/restore_points` or something like that? If the DB is empty those zips will take almost no space at all and it's possible that they will save an operator who was in a hurry and just said "yes" to all questions without really reading them.

Added `mongo/db` backup.

**Review by kwypchlo — APPROVED** [review]

**Review by MSevey — APPROVED** [review]

**Pull request merged**